app/routes/index.tsx

```diff
@@ -1,3 +1,29 @@
+import type { LinksFunction } from "@remix-run/node";
+import { Link } from "@remix-run/react";
+
+import stylesUrl from "~/styles/index.css";
+
+export const links: LinksFunction = () => {
+  return [{ rel: "stylesheet", href: stylesUrl }];
+};
+
 export default function IndexRoute() {
-  return <div>Main Wrapper has app overview</div>;
+  return (
+    <div className="container">
+      <div className="content">
+        <h1>
+          Remix <span>To do list</span>
+        </h1>
+        <nav>
+          <ul>
+            <li>
+              <Link to="todo">To do's</Link>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  );
```

Comment:
This component should hold 4 components like you broke it down, Calendar, Todo, PlanYourDay, PlanYourExpenses... unless you planned to wrap the three components apart from calendar in the `nav`